### PR TITLE
Changing README's to include new set of charts

### PR DESCRIPTION
**Pull request by theburi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,20 @@
...
-In order to install MongoDB DataBase:
+## Charts
 
-```helm upgrade mongodb . --set opsManager.configMap=opsmanager-configmap --set opsManager.secretRef=opsmanager-org-access-key  -n $MONGODB_NAMESPACE --create-namespace -i```
+This repository contains sample HELM charts for different MongoDB products
 
-Where `opsmanager-configmap` and `opsmanager-org-access-key` contain OpsManager connection properties
+| Charts                  | Description                                        |
+|-------------------------|----------------------------------------------------|
+| atlas-operator          | MongoDB Atlas Helm Chart. Install Operator Chart   |
```
Comment (antonlisovenko) [inline README.md:10]:
```suggestion
| atlas-operator          | MongoDB Atlas Helm Chart. Installs Operator Chart   |
```

File: README.md [hunk 2]
```diff
@@ -1,32 +1,20 @@
...
+| atlas-operator-crds     | MongoDB Atlas Collection of CRD's                  |
+|-------------------------|----------------------------------------------------|
+| ent-operator            | MongoDB Enterprise Kubernetes operator. *          |
+| ent-operator-database   | Deploy MongoDB Enterprise DataBase. *              |
+| ent-operator-opsmanager | Deploy MongoDB Enterprise OpsManager. *            |
 
-Helper script could be found at ./helpers/MongoDB-deploy.sh It contains an example that automates MongoDB Deployment using mongocli
+* Charts are in Alpha version and could be used for evaluation purposes only. Please use other methods do deploy these products.
```
Comment (antonlisovenko) [inline README.md:17]: do we need this? Could scare the customers. :)

**Review by antonlisovenko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (theburi) [reply README.md:10]: this is correct. its a call for action

File: README.md (hunk 2 above)
Comment (theburi) [reply README.md]: good point

**Pull request merged**